### ISSUE TITLE
Email login: Add a disclaimer

### ISSUE THREAD

**Issue by davidhedlund** [mozilla/discourse#123]
### Describe the problem and steps to reproduce it:
https://discourse.mozilla.org/ -> Log In / Sign Up
* LOG IN WITH EMAIL -> youremail@example.com (real mail provided of course), SEND EMAIL -> "We sent you a link to sign in. Please check your e-mail and open the provided link in this web browser."

### What happened?
When I open the link provided in the email I get this error.

![screenshot from 2018-02-11 02 12 11](https://user-images.githubusercontent.com/6416073/36068253-0d52fac2-0ed1-11e8-805d-053bf8a14942.png)

### What did you expect to happen?
I should have been logged in.

### Anything else we should know?
There's no Contact link or similar at https://discourse.mozilla.org/, please add this repository to the page to make it clear where people should submit bugs.

**Comment by hmitsch** [comment]
Hi @davidhedlund,

do you have a profile on Mozillians.org? If yes, can you please send us the link to it? We can then have a better look into this issue.

Best regards,
   Henrik

/cc @LeoMcA @flamingspaz @akatsoulas @johngian

**Comment by davidhedlund** [comment]
@hmitsch No I have not. But I have logged in before, see https://discourse.mozilla.org/users/davidhedlund/

**Comment by LeoMcA** [comment]
Hi David,

As the error message describes, you'll need to log in to Discourse using Github from now on.

We're tracking making this experience better here: https://github.com/mozilla-iam/auth0-deploy/issues/166

> There's no Contact link or similar at https://discourse.mozilla.org/, please add this repository to the page to make it clear where people should submit bugs.

There's a contact email listed on https://discourse.mozilla.org/about

**Comment by davidhedlund** [comment]
@LeoMcA Hahaha, what a wonderful avatar!

**Comment by davidhedlund** [comment]
@LeoMcA Why not remove the "LOG IN WITH EMAIL"? There's no use to display it if we cannot use it.

**Comment by LeoMcA** [comment]
> @LeoMcA Hahaha, what a wonderful avatar!

Thanks

> @LeoMcA Why not remove the "LOG IN WITH EMAIL"? There's no need to display it if we cannot use it. It will only continue to cause distrust in the Discourse infrastructure.

*One* can use it, but *you* cannot - because you've logged in with github previously. Users who haven't logged in with github or google previously can continue to log in with email.

Discussion about that isn't a Discourse issue but a wider IAM one, so I'm closing this bug. If you have any further problems authenticating please reach out to us using the contact email listed on https://discourse.mozilla.org/about

**Comment by davidhedlund** [comment]
@LeoMcA Why not add a disclaimer if I fail to login with the mail that reads something like this?:
> You cannot login with mail because you've logged in with GitHub previously.

**Comment by davidhedlund** [comment]
@LeoMcA Great. I changed the title, should the issue be re-opened?

**Comment by edmorley** [comment]
The error message comes from the sso-dashboard, which has just been updated (possibly not deployed yet) to make the reason clearer. See mozilla-iam/sso-dashboard#215.

**Comment by davidhedlund** [comment]
@edmorley Thank you very much.

**Comment by davidhedlund** [comment]
@edmorley Can you re-open this issue?

**Comment by edmorley** [comment]
I'm not involved with the discourse instance and don't have permissions on this GitHub project. However given the message wording has been tweaked, I'm not sure what is left to do here? (The "There's no Contact link" part of the OP should probably be a separate issue etc)

**Comment by davidhedlund** [comment]
@edmorley Ok, thanks again.

**Comment by davidhedlund** [comment]
I still get this error.
![screenshot from 2018-02-11 02 12 11](https://user-images.githubusercontent.com/6416073/36068250-fa7fe5f4-0ed0-11e8-8090-2cf617028c6b.png)

**Comment by edmorley** [comment]
> (possibly not deployed yet)

See:
https://github.com/mozilla-iam/sso-dashboard/pull/215#issuecomment-364690484

**Comment by davidhedlund** [comment]
@edmorley Ok. I just think this bug should be re-opended until this has been fixed.

**Comment by hmitsch** [comment]
@davidhedlund: you are right, let’s reopen the issue. We shall close it, once the tweaked error message lands in production. 

@edmorley: thanks a lot for handling comms on this ticket. This is very helpful! And of course you are right, the tweaked message has only landed on staging. We will test this early next week and hope to deploy before the mid of the week. 

-Henrik

**Comment by davidhedlund** [comment]
@hmitsch I wish you good luck.

**Comment by LeoMcA** [comment]
This shouldn't be an issue any more as most users are now allowed to log in with passwordless even if they've previously logged in with github: https://discourse.mozilla.org/t/mozilla-discourse-release-2019-02-27/36283

**Comment by davidhedlund** [comment]
> This shouldn't be an issue any more as most users are now allowed to log in with passwordless even if they've previously logged in with github: https://discourse.mozilla.org/t/mozilla-discourse-release-2019-02-27/36283

Thanks, there are now four ways to login, not just with GitHub

![screenshot at 2019-03-07 21 53 02](https://user-images.githubusercontent.com/6416073/53988283-68d59780-4123-11e9-9dc6-b9761088c700.png)
